packages/peregrine/lib/talons/Breadcrumbs/useBreadcrumbs.js

```diff
@@ -0,0 +1,51 @@
+import { useMemo } from 'react';
+import { useQuery } from '@apollo/react-hooks';
+
+// Just incase the data is unsorted, lets sort it.
+const sortCrumbs = (a, b) => a.category_level > b.category_level;
+
+// Generates the path for the category.
+const getPath = (path, suffix) => {
+    if (path) {
+        return `/${category_url_path}${suffix}`;
+    }
+
+    // If there is no path this is just a dead link.
+    return '#';
```

Comment:
Is this a more frustrating UX than breadcrumbs that don't actually do anything? Right now we display clickable links like `Home / Shop The Look / Minimalist` where "Shop The Look" is clickable but is a link to the current page (until `category_url_path` is returned from graphql and we start using it.

Should we return `null` or `undefined` instead and have the UI render plain text?

Reply:
Dead links for the sake of the UX looking closer to final state seems ideal here, I'm fine with this.